curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646e3d8a42296d01b0304b18.md

```diff
@@ -0,0 +1,211 @@
+    />
+    <path
+      d="M2230 1855 l0 -155 315 0 315 0 0 155 0 155 -315 0 -315 0 0 -155z"
+    />
+    <path
+      d="M3340 1855 l0 -155 315 0 315 0 0 155 0 155 -315 0 -315 0 0 -155z"
+    />
+    <path
+      d="M1117 1383 c-4 -3 -7 -78 -7 -165 l0 -158 320 0 320 0 0 165 0 165 -313 0 c-173 0 -317 -3 -320 -7z"
+    />
+    <path
+      d="M2230 1225 l0 -165 315 0 315 0 0 165 0 165 -315 0 -315 0 0 -165z"
+    />
+    <path
+      d="M3347 1383 c-4 -3 -7 -78 -7 -165 l0 -158 315 0 315 0 -2 163 -3 162 -306 3 c-168 1 -308 -1 -312 -5z"
+    />
+  </g>
+</svg>
+      </div>
+      <div class="divider"></div>
+      <div class="dropdown-container">
+        <select name="date-options" id="date-options">
+          <option value="dd-mm-yyyy">Day, Month, Year</option>
+          <option value="yyyy-mm-dd">Year, Month, Day</option>
+          <option value="mm-dd-yyyy-h-mm">
+            Month, Day, Year, Hours, Minutes
+          </option>
+        </select>
+        <img src="an image link here" class="select-icon"/>
+      </div>
+      <p id="current-date"></p>
+    </main>
+  </body>
+  <script src="script.js"></script>
+</html>
+```
+
+```css
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box
+}
+:root {
+  --dark-grey: #0a0a23;
+  --white: #ffffff;
+  --yellow: #f1be32;
+}
+
+body{
+  background-color: var(--dark-grey);
+  color: var(--white);
+}
+
+.title-container{
+  margin: 10px 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 8px;
+}
+
+.title {
+  font-size: 2.5rem;
+}
+
+.date-svg{
+  width: 30px;
+  height: 30px;
+}
+
+.divider {
+  margin: auto;
+  width: 150px;
+  height: 10px;
+  background-color: var(--yellow);
+}
+
+.dropdown-container {
+  width: 100%;
+  margin: 40px auto;
+  position: relative;
+}
+
+select {
+  display: block;
+  margin: 20px auto;
+  height: 50px;
+  font-size: 100%;
+  font-weight: bold;
+
+  cursor: pointer;
+  background-color: var(--white);
+  border: none;
+  color: var(--dark-grey);
+  appearance: none;
+  padding: 10px 38px 10px 10px;
+}
+
+.select-icon{
+  position: absolute;
+  top: 4px;
+  right: 4px;
+  pointer-events: none;
+  width: 30px;
+  height: 36px;
+  padding-left: 4px;
+}
+
+#current-date{
+  font-size: 2rem;
+  text-align:center
+}
+```
+
+```js
+const currentDateParagraph = document.getElementById("current-date")
+const dateOptionsSelectElement = document.getElementById("date-options")
+
+const date = new Date();
+const day = date.getDate();
+const month = date.getMonth() + 1;
+const year = date.getFullYear();
+const hours = date.getHours();
+const minutes = date.getMinutes();
+
+const formattedDate = `${day}-${month}-${year}`
+currentDateParagraph.textContent = formattedDate
+
+dateOptionsSelectElement.addEventListener("change", ()=>{
+  switch(dateOptionsSelectElement.value){
+--fcc-editable-region--
+
+--fcc-editable-region--
+    default: 
+      currentDateParagraph.textContent = formattedDate
+    break
```

Comment:
We should remove this break statement since it wasn't mentioned in the previous step and it is not needed since this is the default case 

```suggestion

```